src/client/components/CloseButton/CloseButton.tsx

```diff
@@ -3,7 +3,7 @@ import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
 import { Cross } from '../icons/Cross'
 
-const Button = styled.button`
+export const StyledCloseButton = styled.button`
```

Comment:
nit: Should we skip the Styled suffix for styled components? To me it feels a bit unnecessary and maybe confusing since we don't use it everywhere. I have probably written a few myself though :D

Reply:
Good one! I think I did it here just to be sure I hit the right button with the styling, but will fix 👍

Reply:
If you don't mine I gonna add more fuel to the discussion ⛽ 

I would also prefer removing the _Styled_ prefix on this case, ending with something like `export const CloseButton = styled.button`. But what do you guys think it's a good name in the following scenario?

```javascript
const Button = styled.button``

const <insert-component-name-here> = styled(Button)``
```

I usually use the _Styled_ prefix on such cases, so I'd like to know your opinions about it.

Reply:
I agree @guilhermespopolin, I am also a bit unsure about the conventions in those cases :)